frontend/packages/dev-console/src/components/add/AddCardItem.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { useUserSettings } from '@console/shared';
+import { SimpleListItem, Title, Text } from '@patternfly/react-core';
+import { ResolvedExtension, AddAction } from '@console/dynamic-plugin-sdk';
+import { navigateTo, resolvedHref } from '../../utils/add-page-utils';
+import './AddPage.scss';
+
+type AddCardItemProps = {
+  action: ResolvedExtension<AddAction>;
+  namespace: string;
+};
+
+const AddCardItem: React.FC<AddCardItemProps> = ({
+  action: {
+    properties: { id, label, icon, href, description },
+  },
+  namespace,
+}) => {
+  const [showDetails] = useUserSettings('console.addPage.showDetails', 'show', true);
+  const actionIcon = (): JSX.Element => {
+    if (typeof icon === 'string') {
+      return <img className="odc-add-page__card-item-header-icon" src={icon} alt={label} />;
```

Comment:
You can hide this from screen readers (aria-hidden="true") if this has just the same information (alt text) then the title after this.

```suggestion
      return <img className="odc-add-page__card-item-header-icon" src={icon} alt={label} aria-hidden="true" />;
```